rebasehelper/application.py

```diff
@@ -344,11 +341,11 @@ def _check_results_dir(results_dir):
             logger.warning("Results directory '%s' exists, removing it", os.path.basename(results_dir))
             shutil.rmtree(results_dir)
         os.makedirs(results_dir)
-        os.makedirs(os.path.join(results_dir, settings.REBASE_HELPER_LOGS))
-        os.makedirs(os.path.join(results_dir, 'old-build'))
-        os.makedirs(os.path.join(results_dir, 'new-build'))
-        os.makedirs(os.path.join(results_dir, 'checkers'))
-        os.makedirs(os.path.join(results_dir, 'rebased-sources'))
+        os.makedirs(os.path.join(results_dir, constants.REBASE_HELPER_LOGS_DIR))
+        os.makedirs(os.path.join(results_dir, constants.REBASE_HELPER_OLD_BUILD_DIR))
+        os.makedirs(os.path.join(results_dir, constants.REBASE_HELPER_NEW_BUILD_DIR))
+        os.makedirs(os.path.join(results_dir, constants.REBASE_HELPER_CHECKERS_DIR))
+        os.makedirs(os.path.join(results_dir, constants.REBASE_HELPER_REBASED_SOURCES_DIR))
```

Comment:
Maybe it would be worth it in this case to import all the constants using wildcard (`from rebasehelper.constants import *`). Also `REBASE_HELPER_` prefix is kind of unnecessary I would say.